tests/drivers/flash_simulator/prj.conf

```diff
@@ -1,5 +1,6 @@
 CONFIG_ZTEST=y
 CONFIG_FLASH=y
+CONFIG_FLASH_ERASE_VALUE=y
```

Comment:
```suggestion
CONFIG_FLASH_GET_PARAMETERS_API=y
```

Reply:
fix in proper commit

Reply:
Agree, this doesn't belong in this commit.